### ISSUE TITLE
Setting src and dest to same path == Misery

### ISSUE THREAD

**Issue by ryascl** [krasimir/techy#35]
I tried using the same path for --src and --dest to get side-by-side rendering. It created a path so long in Windows I spent half an hour figuring out how to delete the files. <facepalm>

**Comment by krasimir** [comment]
Well ... the idea of the module is to produce output in a directory that is later uploaded to the server. If `src` and `dest` are identical then you'll upload your source files too. What you meant by `side-by-side` rendering?

**Comment by ryascl** [comment]
Yes, you are right. The intended function of src and dest is better than rendering into the same directory. I was just trying it out.
The app could fail with error if an output file/dir is received as input, to protect dumb users like me. Not a big deal.

**Comment by krasimir** [comment]
Well, I did the same while I was developing the module :) And indeed it's difficult to delete the folders under Windows :D
